### PR TITLE
Text changes to Home, Relationships and Actor Definitions pages (FHIR-46797, FHIR-46798)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-46797](https://jira.hl7.org/browse/FHIR-46797)
* Changed required text in 1.1 Introduction

Fix for [FHIR-46789](https://jira.hl7.org/browse/FHIR-46798)
* Changed required text in 1.2 Project Background
* Changed page name from 'Relationship to other IGs' to 'Relationships to other Specifications'

### CODE REVIEW

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ The focus AU eRequesting Release 1 (R1) is support of pathology and medical imag
...
 
-The AU eRequesting project includes:
-- AU eRequesting Data for Interoperability
-- AU eRequesting FHIR Implementation Guide
+The Sparked AU FHIR Accelerator includes:
+- AU eRequesting Data for Interoperability - clinician focussed data requirements
+- AU eRequesting FHIR Implementation Guide - HL7 Australia technical data access and exchange specification
 
 The [Australian eRequesting Data for Interoperability (AUeReqDI)](https://sparked.csiro.au/index.php/sparked-products-resources/auereqdi/auereqdi-release-1/) is focused on an agreement of the minimum data required to support standardised eRequesting within the Australian health context, and forms a common language foundation that allows systems to exchange semantically accurate data for eRequests. AUeReqDI outputs form a set of data requirements to be considered and referred to as part of the development and definition of AU eRequesting.
```
Comment (dbojicic) [inline input/pagecontent/index.md:18]: Commenting here to get an inline comment. 
Please update AU Core Placer description in 1.4 AU eRequesting Actors as well.

File: input/pagecontent/index.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ The focus AU eRequesting Release 1 (R1) is support of pathology and medical imag
...
```
Comment (dbojicic) [inline input/pagecontent/index.md:17]: Commenting here to get an inline comment. Please update AU eRequesting Placer description in the Conformance page.

File: input/fsh/actors.fsh [hunk 3]
```diff
@@ -5,7 +5,7 @@ Usage: #definition
 * name = "AUeRequestingPlacer"
 * title = "AU eRequesting Placer"
 * status = #draft
-* description = "The AU eRequesting Placer is a system that is responsible for initiating and creating diagnostic service requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request."
+* description = "The AU eRequesting Placer is a system that is responsible for creating diagnostic requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request."
```
Comment (dbojicic) [inline input/fsh/actors.fsh:8]: I don't think this implements the [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.

File: input/pagecontent/actors.md [hunk 4]
```diff
@@ -2,11 +2,11 @@ The following ActorDefinitions define the systems that plays a role in data exch
 
 - [AU eRequesting Placer actor](ActorDefinition-au-erequesting-actor-placer.html)
 
-The AU eRequesting Placer is a system that is responsible for initiating and creating diagnostic service requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.  
+The AU eRequesting Placer is a system that is responsible for creating diagnostic requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.
```
Comment (dbojicic) [inline input/pagecontent/actors.md:5]: I don't think this implements the [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.

Comment (dbojicic) [inline input/pagecontent/actors.md:5]: Commenting here to get an inline comment. 
Please update the AU eRequesting Placer actor description in 6 Capability Statements section, and in AU eRequesting Placer CapabilityStatement.

File: input/fsh/actors.fsh [hunk 5]
```diff
@@ -19,7 +19,7 @@ Usage: #definition
 * name = "AUeRequestingFiller"
 * title = "AU eRequesting Filler"
 * status = #draft
-* description = "The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process."
+* description = "The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic request fulfilments so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process."
```
Comment (dbojicic) [inline input/fsh/actors.fsh:22]: I don't think this implements [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.

File: input/pagecontent/actors.md [hunk 6]
```diff
@@ -2,11 +2,11 @@ The following ActorDefinitions define the systems that plays a role in data exch
...
+The AU eRequesting Placer is a system that is responsible for creating diagnostic requests. This system connects to enable requesters to provide the necessary information to create a request, ensuring that required data is accurately collected and provided to service providers with sufficient information for them to fulfil the request.  
 
 - [AU eRequesting Filler actor](ActorDefinition-au-erequesting-actor-filler.html)
 
-The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process.
+The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic request fulfilments so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process.
```
Comment (dbojicic) [inline input/pagecontent/actors.md:9]: I don't think this implements [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.

File: sushi-config.yaml [hunk 7]
```diff
@@ -53,7 +53,7 @@ pages:
     future.md:
       title: Future of AU eRequesting
     relationship.md: 
-      title: Relationship with other IGs
+      title: Relationships to other Specifications
```
Comment (dbojicic) [inline sushi-config.yaml:56]: This aligns with the JIRA resolution but is inconsistent with other pages that use title case. Can you please check with @michael-wilson-au if we can change to 'Relationships to Other Specifications' to align with convention used throughout the IG? I think we may keep as is in this PR and then raise a TC in JIRA to fix.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/pagecontent/actors.md (hunk 4 above)
Comment (JayMurdoch) [reply input/pagecontent/actors.md:5]: [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) moved to PR covering consolidation of actor definitions.

File: input/pagecontent/actors.md (hunk 6 above)
Comment (JayMurdoch) [reply input/pagecontent/actors.md:9]: [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) moved to PR covering consolidation of actor definitions.

File: input/pagecontent/index.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/index.md:17]: [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) and [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) moved to PR covering consolidation of actor definitions.

File: sushi-config.yaml (hunk 7 above)
Comment (JayMurdoch) [reply sushi-config.yaml:56]: As discussed in IG triage with @michael-wilson-au, amended to reflect convention used throughout IG.

File: input/fsh/actors.fsh (hunk 3 above)
Comment (JayMurdoch) [reply input/fsh/actors.fsh:8]: [FHIR-46799](https://jira.hl7.org/browse/FHIR-46799) moved to PR covering consolidation of actor definitions.

File: input/fsh/actors.fsh (hunk 5 above)
Comment (JayMurdoch) [reply input/fsh/actors.fsh:22]: [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) moved to PR covering consolidation of actor definitions.

**Pull request merged**